src/parcsr_ls/par_multi_interp.c

```diff
@@ -320,13 +322,15 @@ hypre_BoomerAMGBuildMultipassHost( hypre_ParCSRMatrix  *A,
    }
 
    n_coarse_offd = 0;
-   n_SF_offd = 0;
 #ifdef HYPRE_USING_OPENMP
+   n_SF_offd = 0;
    #pragma omp parallel for private(i) reduction(+:n_coarse_offd,n_SF_offd) HYPRE_SMP_SCHEDULE
 #endif
    for (i = 0; i < num_cols_offd; i++)
       if (CF_marker_offd[i] == 1) { n_coarse_offd++; }
+#ifdef HYPRE_USING_OPENMP
       else if (CF_marker_offd[i] == -3) { n_SF_offd++; }
```

Comment:
I thought this line should always happen, regardless of turning on OpenMP support. Let's double-check?

Reply:
I didn't check to see who wrote this code, so I wanted to be careful not to delete something that was meant to be kept around.  Please feel free to delete if you know the situation better than me.  Thanks!

Reply:
If you remove the `HYPRE_USING_OPENMP` guard, the compiler warnings will come back unless you get rid of the `n_SF_offd++` stuff altogether (or comment it out some other way).  Just FYI.

Reply:
I don't know how this OPENMP got in there, but it shouldn't be.

Reply:
I added the OPENMP guard because that was the only context where it was being used.  The warning messages were all about variables being set but not used (in a `pragma`).

Reply:
@ulrikeyang Are we good leaving this one as is then?  Thanks!

Reply:
Okay, done.